src/catalog/src/builtin.rs

```diff
@@ -21,6 +21,13 @@
 //!
 //! More information about builtin system tables and types can be found in
 //! <https://materialize.com/docs/sql/system-catalog/>.
+//!
+//! Several of the `pg_catalog` views have additional, "peeled" versions, which
+//! - don't check `mz_schemas.database_id IS NULL OR d.name = pg_catalog.current_database()`,
+//!   in order to make these views indexable;
+//! - have the database name as an extra column, so that downstream views can check it against
+//!  `current_database()`.
+//! For example, the peeled version of `pg_class` is `pg_class_all_databases`.
```

Comment:
I don't think this needs to be a module level comment, I think it would be more helpful and less distracting if you put it directly over the peeled views.

Reply:
That's subjective though so feel free to ignore.

Reply:
Ok, I've changed it.